lib/plugins/Label_80.test.ts

```diff
@@ -135,7 +135,7 @@ test('decodes Label 80 variant 2', () => {
   expect(decodeResult.formatted.items[14].type).toBe('fuel_on_board');
   expect(decodeResult.formatted.items[14].code).toBe('FOB');
   expect(decodeResult.formatted.items[14].label).toBe('Fuel On Board');
-  expect(decodeResult.formatted.items[14].value).toBe('N009414');
+  expect(decodeResult.formatted.items[14].value).toBe('NaN');
```

Comment:
This is for the better, right?

Reply:
should probably just not push the item at all if its NaN, right?

Reply:
That's actually what I think. I'll make the change